tests/acceptance/learnergroups-test.js

```diff
@@ -85,7 +82,9 @@ module('Acceptance | Learner Groups', function(hooks) {
     assert.equal(page.learnerGroupList.groups[0].title, 'None');
 
     await page.programYearFilter.filter(1);
+    //await pauseTest();
```

Comment:
Kindly remove this awesome piece of debugging code!

Reply:
removed.